src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -241,7 +261,13 @@ const SummaryBar = props => {
               {!weeklySummary ? (
                 <div className="border-red col-4 bg--white-smoke no-gutters" align="center">
                   <div className="py-1"> </div>
-                  {matchUser ? (
+                  {matchUser ||
+                  hasPermission(
+                    authenticateUserRole,
+                    'submitWeeklySummaryForOthers',
+                    roles,
+                    authenticateUserPermission,
+                  ) ? (
```

Comment:
This condition seems to be the same as the one above in lines 187 through 192. Let's assign this boolean to a const before the return and use it in both places.